applications/MeshingApplication/custom_utilities/mmg_utility.h

```diff
@@ -213,11 +213,11 @@ class MmgUtility
         /* We print the original model part */
         if (mEchoLevel > 0)
         {
-           std::cout << "//---------------------------------------------------//" << std::endl;
-           std::cout << "//---------------------------------------------------//" << std::endl;
-           std::cout << "//---------------  BEFORE REMESHING   ---------------//" << std::endl;
-           std::cout << "//---------------------------------------------------//" << std::endl;
-           std::cout << "//---------------------------------------------------//" << std::endl << std::endl;
+            std::cout << "//---------------------------------------------------//" << std::endl;
+            std::cout << "//---------------------------------------------------//" << std::endl;
+            std::cout << "//---------------  BEFORE REMESHING   ---------------//" << std::endl;
+            std::cout << "//---------------------------------------------------//" << std::endl;
+            std::cout << "//---------------------------------------------------//" << std::cout << std::endl;
```

Comment:
I'm referring to line 220: `std::cout << "//--...--//" << std::cout << std::endl;`
Why do you need two `std::cout` there?